packages/block-editor/src/components/link-control/index.js

```diff
@@ -133,15 +139,33 @@ function LinkControl( {
 		withCreateSuggestion = true;
 	}
 
+	const [ settingsOpen, setSettingsOpen ] = useState( false );
```

Comment:
We need the local state as a fallback in case that the preference isn't supplied. I considered using a local state as the value passed as the setting but I realised that is also no good because the editor might not pass the settings _at all_ so we cannot rely on their presence.